src/osemgrep/reporting/Nosem.ml

```diff
@@ -69,11 +70,96 @@ let _nosem_previous_line_re =
 (* Entry point *)
 (*****************************************************************************)
 
-(* TODO: probably need to pass some flags from the CLI *)
-let process_ignores (out : Out.cli_output) : Out.cli_output =
-  (* TODO: port nosemgrep.py, filter in out.matches the
-     matches where the match has a corresponding nosem: annotation in
-     the corresponding file, and add new errors if nosem: has the wrong
-     format in the file.
-  *)
-  out
+let rule_match_nosem ~strict (rule_match : Out.cli_match) :
```

Comment:
you can move this in the Helpers section (it's not an entry point of the module).